packages/api/prisma/seed.ts

```diff
@@ -119,6 +121,21 @@ async function run() {
     })),
   });
 
+  const language = await client.language.create({
+    data: {
+      code: 'en',
+      name: 'English',
+    },
+  });
+
+  await client.gloss.createMany({
+    data: wordData.map((word) => ({
+      wordId: word.id,
+      languageId: language.id,
+      gloss: word.english,
+    })),
+  });
```

Comment:
This seed script now also inserts English glosses to use as the default reference gloss

Reply:
I wondered if we would use that data. 👍🏽